test_runner/performance/test_bulk_insert.py

```diff
@@ -4,12 +4,6 @@
 from fixtures.benchmark_fixture import MetricReport, ZenithBenchmarker
 from fixtures.compare_fixtures import PgCompare, VanillaCompare, ZenithCompare
 
-pytest_plugins = (
-    "fixtures.zenith_fixtures",
-    "fixtures.benchmark_fixture",
-    "fixtures.compare_fixtures",
-)
```

Comment:
Conftest mentions only zenith_fixtures, and benchmark_fixture, probably compare_fixtures should be added too, but everything works even without it (as I can see all the checks are green)

Reply:
It's there. Maybe you're looking at a stale main branch :)